### ISSUE TITLE
Substitute values in summary rows

### ISSUE THREAD

**Issue by RemiDumas** [rstudio/gt#1878]
## Prework

- [ ] Read and abide by **gt**'s [code of conduct](https://www.contributor-covenant.org/version/2/1/code_of_conduct.html) and [contributing guidelines](https://github.com/rstudio/gt/blob/master/.github/CONTRIBUTING.md).
- [ ] Search for duplicates among the [existing issues](https://github.com/rstudio/gt/issues) (both open and closed).

## Proposal

`sub_values()` function only allows to substitute values in body rows. It'll be useful to hide an entire column keeping the column label and the empty cells.

**Comment by olivroy** [comment]
I also agree this would be useful. Please provide more details of your use case!

As a workaround, you can define a function that substitute your desired values (See example in #1440)
